arch/cortex-m4/src/lib.rs

```diff
@@ -3,41 +3,51 @@
 #![crate_name = "cortexm4"]
 #![crate_type = "rlib"]
 #![no_std]
+#![feature(naked_functions)]
```

Comment:
Why do we need to add this feature? I don't see any naked functions added in this crate

Reply:
My bad, this was leftover from a previous attempt which didn't use associated constants. Wondering why I didn't get a warning about an unused feature here?